src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {
+        person.age
+        && (
+          <p className="Person__age">
+            {`I am ${person.age}`}
+          </p>
+        )
+      }
+    <p className="Person__partner">
```

Comment:
```suggestion
    </h2>
    
    {person.age && (
      <p className="Person__age">
        {`I am ${person.age}`}
      </p>
    )}
     
    <p className="Person__partner">
```
Fix formatting to make your code easier to read